sema/subyt/sinks.py

```
@@ -74,17 +75,19 @@ def add(
 
 
 class SingleFileSink(Sink):
-    def __init__(self, file_path: str, force_output: bool = False):
+    def __init__(self, path_name: str, force_output: bool = False):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)